### DISCUSSION TITLE
JetStream with teams initialisation

### DISCUSSION THREAD

**Discussion by freasy** [TiagoSilvaPereira/vemto-issues#365] [Ideas & Suggestions]
I want the jetstream teams functionality. With the current version it initializes the basic jetstream and overrides the usermodel according to database configurations.

When i use `jetstream:install livewire --teams` afterwards to get teams functions, it overrides the vemto user model.
In addition the teams table is not visible in the database planer.

We we get the option to enable jetstream teams capability and reflect it in the database?

**Comment by TiagoSilvaPereira** [comment]
Hi @freasy thank you for your suggestion, we'll add this option in the future
